site/app/controllers/forum/ForumController.php

```diff
@@ -575,6 +581,7 @@ public function alterPost($modifyType){
                 }
             } else {
                 if($status_edit_thread && $status_edit_post) {
+                    $type = "Thread and Post";
```

Comment:
Thoughts on using an enum here instead (possibly forum alter intent)? I know php doesn't have native support. Possibly introduce an abstract class with const members? Just a thought, would like to get your opinion. I'm personally not the biggest fan of the string comparison down below.

Reply:
I also do not like this string comparison, but I feel like this function in general should be split up into alterThread and alterPost since there a multiple places where there is a logic control block checking if it is a thread or a post.